rush/rush/src/actions/InstallAction.ts

```diff
@@ -104,12 +104,12 @@ export default class InstallAction extends CommandLineAction {
       actionVerb: 'install',
```

Comment:
I am going to publish these locally, since we have rush autopublishing turned off right now. There were a few other commits missing a changefile because of that also, so I will make sure those get documented